icu4j/main/classes/core/src/com/ibm/icu/text/RuleBasedNumberFormat.java

```diff
@@ -275,7 +275,7 @@
  *   <tr>
  *     <td style="width: 5%; vertical-align: top;"></td>
  *     <td style="width: 8%; vertical-align: top;">x.0:</td>
- *     <td valign="top">The rule is a <em>master rule</em>. If the full stop in
+ *     <td valign="top">The rule is a <em>primary rule</em>. If the full stop in
```

Comment:
My previous comments about rbnf.h applies to all of the RBNF Java code too. The documentation and changes should be kept in sync. It should be "default" and not "master" nor "primary".

Reply:
Done. Thx.

Reply:
OK, I think I've handled this better now. Thanks for your patience explanation.